configure.ac

```diff
@@ -305,7 +305,7 @@ AC_MSG_RESULT([$enable_msabi_support])
 LIBLZMA=
 AC_MSG_CHECKING([whether to support LZMA-compressed symbol tables])
 AC_ARG_ENABLE(minidebuginfo,
-AS_HELP_STRING([--enable-minidebuginfo], [Enables support for LZMA-compressed symbol tables]),, [enable_minidebuginfo=auto])
+AS_HELP_STRING([--enable-minidebuginfo], [Enables support for LZMA-compressed symbol tables]),, [enable_minidebuginfo=no])
```

Comment:
Why? A lot of existing builds use the autodetection. They will break with this change. If your build target audetects support for LZMA-compressed symbol tables and you explicitly don't want them (an exceptional situation), it's better that you explicitly opt out instead of forcing the default case to explicitly opt in.

Reply:
see the referenced issue #123